s3/src/main/scala/org/apache/pekko/stream/connectors/s3/impl/SplitAfterSize.scala

```diff
@@ -38,43 +38,46 @@ import scala.annotation.tailrec
   def apply[I, M](minChunkSize: Int,
       maxChunkSize: Int)(in: Flow[I, ByteString, M]): SubFlow[ByteString, M, in.Repr, in.Closed] = {
     require(minChunkSize < maxChunkSize, "the min chunk size must be smaller than the max chunk size")
-    in.via(insertMarkers(minChunkSize, maxChunkSize)).splitWhen(_ == NewStream).collect { case bs: ByteString => bs }
+    in.via(insertMarkers(minChunkSize, maxChunkSize))
+      .splitWhen(_ == NewStream)
+      .collect { case bs: ByteString => bs }
   }
 
   private case object NewStream
 
-  private def insertMarkers(minChunkSize: Long, maxChunkSize: Int) = new GraphStage[FlowShape[ByteString, Any]] {
-    val in = Inlet[ByteString]("SplitAfterSize.in")
-    val out = Outlet[Any]("SplitAfterSize.out")
-    override val shape = FlowShape.of(in, out)
+  private def insertMarkers(minChunkSize: Long, maxChunkSize: Int): GraphStage[FlowShape[ByteString, Any]] =
```

Comment:
Adding an explicit type annotation, otherwise, it won't compile locally.

Reply:
This is compiling fine for me locally and its also compiling fine in CI, do you have something unique with your setup?

Reply:
@He-Pin Did you manage to look into this? Otherwise the PR looks fine from my side.